models/users-models.js

```diff
@@ -1,6 +1,17 @@
 const db = require('../db/connection');
+
 exports.selectUsers = () => {
   return db.query('SELECT username FROM users').then((users) => {
     return users.rows;
   });
 };
+
+exports.selectUserById = (username) => {
+  return db
+    .query('SELECT username, name, avatar_url FROM users WHERE username = $1', [
```

Comment:
Don't forget semicolons 😄

Reply:
I have gone through all of my models now!